pyproject.toml

```diff
@@ -25,6 +25,7 @@ dynamic = ["version"]
 dependencies = [
     "fitsio",
     "numpy",
+    "astropy",
```

Comment:
I'm not entirely happy with the dependency on astropy, since astropy routinely breaks projects. Maybe we can refactor the FITS reading to use fitsio later.

Reply:
I am reading the new entries with fitsio but left the old ones with astropy for now.